tests/tests_geomstats/test_euclidean.py

```diff
@@ -1,373 +1,442 @@
 """Unit tests for the Euclidean space."""
+import math
+import random
 
 import geomstats.backend as gs
-import geomstats.tests
-from geomstats.geometry.euclidean import Euclidean
-
-
-class TestEuclidean(geomstats.tests.TestCase):
-    def setup_method(self):
-        gs.random.seed(1234)
-
-        self.dimension = 2
-        self.space = Euclidean(self.dimension)
-        self.metric = self.space.metric
-
-        self.n_samples = 3
-
-        self.one_point_a = gs.array([0.0, 1.0])
-        self.one_point_b = gs.array([2.0, 10.0])
-        self.n_points_a = gs.array([[2.0, 1.0], [-2.0, -4.0], [-5.0, 1.0]])
-        self.n_points_b = gs.array([[2.0, 10.0], [8.0, -1.0], [-3.0, 6.0]])
-
-    def test_random_point_and_belongs(self):
-        point = self.space.random_point()
-        result = self.space.belongs(point)
-        expected = True
-
-        self.assertAllClose(result, expected)
-
-    def test_is_tangent(self):
```

Comment:
yeah.. this will be in the next improvements. I removed it because, we will be adding again. Also I am thinking to_tangent is identity for only points on manifold. not necessarily for every point because. If we consider symmetric matrices, not very matrix is symmetric matrix.

Reply:
I removed it because, I didn't wanna spend on time on something I will delete it and to_tangent, is_tangent is already being tested in to_tangent_is_tangent way. so codecov is not being changed.

Reply:
ok